src/Features/Core/Portable/InlineHints/InlineHint.cs

```diff
@@ -28,6 +29,22 @@ namespace Microsoft.CodeAnalysis.InlineHints
             Span = span;
             DisplayParts = displayParts;
             _getDescriptionAsync = getDescriptionAsync;
+            ReplacementText = null;
+        }
```

Comment:
have the above function just delegate into the new function by using `: this(the args)`

Reply:
fixed